### PR TITLE
Mixpanel integration

### PR DESCRIPTION
**Pull request by Tolsa**

### CODE REVIEW

File: _includes/lumber/integrations.md [hunk 1]
```diff
@@ -29,3 +29,17 @@ Here are the complete list of available options to customize your Stripe integra
...
+
+## Mixpanel
+<span class="l-category l-category__pro">Premium feature</span>
+
+The Mixpanel integration for Forest allows you to fetch Mixpanel's events and display them at a record level. 
+
+To benefit from Mixpanel integration, you need to add the package `mixpanel-data-export` before going further.
+
+Then, add the following code to your `app.js` file. In our example we will map the `customers.email` with the data coming from Mixpanel. You may replace by your own relevant collection(s):
+<div data-github-gist="Tolsa/ec20fa891cd9c869428335032a5403f7"></div>
```
Comment (SeyZ) [inline _includes/lumber/integrations.md]: Your gist has bad indentation

Comment (SeyZ) [reply _includes/lumber/integrations.md]: You should also explain what is the `customProperties` option

File: _includes/rails/integrations.md [hunk 2]
```diff
@@ -29,3 +29,18 @@ Here are the complete list of available options to customize your Stripe integra
...
 
+## Mixpanel
+<span class="l-category l-category__pro">Premium feature</span>
+
+The Mixpanel integration for Forest allows you to fetch Mixpanel's events and display them at a record level. 
+
+To benefit from Mixpanel integration, you need to add the `gem 'mixpanel_client'` to your Gemfile.
+
+Then, add the following code to your initializer. In our example we will map the `Customer.email` with the data coming from Mixpanel. You may replace by your own relevant collection(s):
+<div data-github-gist="Tolsa/04477f73c4af9edea76f655f018efc1f"></div>
```
Comment (SeyZ) [inline _includes/rails/integrations.md]: Same, you should explain what is `custom_properties`

File: _includes/lumber/integrations.md (hunk 1 above)
Comment (Tolsa) [reply _includes/lumber/integrations.md]: @SeyZ thanks for the feedback ! Gist is updated now and I will add soon the customProperties part

File: _includes/express-mongoose/integrations.md [hunk 3]
```diff
@@ -85,3 +85,48 @@ app.use(Liana.init({
 </div>
 
 ![Stripe 1](/public/img/stripe-1.png)
+
+## Mixpanel
+<span class="l-category l-category__pro">Premium feature</span>
+
+The Mixpanel integration for Forest allows you to fetch Mixpanel's events and display them at a record level.
```
Comment (louisteenbrink) [inline _includes/express-mongoose/integrations.md]: The Mixpanel integration allows you to fetch Mixpanel's events and display them at a record level into Forest.

File: _includes/express-sequelize/integrations.md [hunk 4]
```diff
@@ -85,3 +85,48 @@ app.use(Liana.init({
 </div>
 
 ![Stripe 1](/public/img/stripe-1.png)
+
+## Mixpanel
+<span class="l-category l-category__pro">Premium feature</span>
+
+The Mixpanel integration for Forest allows you to fetch Mixpanel's events and display them at a record level.
```
Comment (louisteenbrink) [inline _includes/express-sequelize/integrations.md]: The Mixpanel integration allows you to fetch Mixpanel's events and display them at a record level into Forest.

File: _includes/lumber/integrations.md [hunk 5]
```diff
@@ -29,3 +29,17 @@ Here are the complete list of available options to customize your Stripe integra
 | mapping               | string          | Indicates how to reconcile your Customer data from your Stripe account and your collection/field from your database. Format must be model_name.stripe_customer_id_field
 | stripe                | library         | The official [Node.js Stripe library](https://github.com/stripe/stripe-node) NPM package.
 {:.markdown-table}
+
+## Mixpanel
+<span class="l-category l-category__pro">Premium feature</span>
+
+The Mixpanel integration for Forest allows you to fetch Mixpanel's events and display them at a record level.
```
Comment (louisteenbrink) [inline _includes/lumber/integrations.md]: The Mixpanel integration allows you to fetch Mixpanel's events and display them at a record level into Forest.

File: _includes/rails/integrations.md [hunk 6]
```diff
@@ -29,3 +29,18 @@ Here are the complete list of available options to customize your Stripe integra
 | mapping               | string          | Indicates how to reconcile your Customer data from your Stripe account and your collection/field from your database. Format must be model_name.stripe_customer_id_field
 {:.markdown-table}
 
+## Mixpanel
+<span class="l-category l-category__pro">Premium feature</span>
+
+The Mixpanel integration for Forest allows you to fetch Mixpanel's events and display them at a record level.
```
Comment (louisteenbrink) [inline _includes/rails/integrations.md]: The Mixpanel integration allows you to fetch Mixpanel's events and display them at a record level into Forest.

**Pull request merged**